src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  pressed: boolean,
+  key: null | string
```

Comment:
I believe you just need one variables in state
```suggestion
  pressedKey: string,
```